### ISSUE TITLE
Resolve CRAN check 'NOTE' for 1.10.17-11

### ISSUE THREAD

**Issue by gothub** [ropensci/redland-bindings#82]
The latest [CRAN checks](https://cran.r-project.org/web/checks/check_results_redland.html) uncovered the following:
```
Version: 1.0.17-11
Check: top-level files
Result: NOTE
    ‘configure’: /bin/bash is not portable
Flavors: r-devel-linux-x86_64-fedora-clang, r-devel-linux-x86_64-fedora-gcc
```

**Comment by gothub** [comment]
Testing on macos with testthat 3e:
- devtools::check() now requires that 'checkbashisms' be installed
  - a check is made of the ./configure script to verify that it is POSIX compliant
     - have to use "#!/bin/sh" instead of "#!/bin/bash"
  - on macos, this is needed: brew install checkbashisms
  - on ubuntu, sudo apt install devscripts, to install checkbashisms

**Comment by gothub** [comment]
Updated 'configure' script so that it passes `devtools::check()` checks, esp. the check that calls 'checkbashims', which checks that bash scripts only use POSIX compliant statements that are compatible across OS types. Fixed in commit b42ee195197a1a3077f85e2de3711e1659ff4bbe
